phys/module_sf_noahdrv.F

```diff
@@ -3305,6 +3306,7 @@ SUBROUTINE lsm_mosaic(DZ8W,QV3D,P8W3D,T3D,TSK,                      &
 #endif
       
                 IF (rdlai2d) THEN
+                   if(lai(i,j).eq.0.0) lai(i,j) = 0.01 
```

Comment:
@barlage @smileMchen 
Mike,
Should your proposed mod
```
if(shdfac == 0.0) lai(i,j) = 0.0
if(shdfac > 0.0 .and. lai(i,j) == 0.0) lai(i,j) = 0.01
```
also be substituted in this second location?

Reply:
@davegill 
Dave,
LAI cannot be zero, otherwise it will trigger segmentation fault.

Reply:
My modifications do not need to be included in either place because they are not addressing/solving the current issue of the pull request, apparently. I have no answer for why the model seg faults (in Noah) for LAI=0.